e2e/test/scenarios/admin-2/settings.cy.spec.js

```diff
@@ -1156,7 +1156,9 @@ describe("notifications", { tags: "@external" }, () => {
         "DELETE",
         `${WEBHOOK_TEST_HOST}/api/session/${WEBHOOK_TEST_SESSION_ID}/requests`,
         { failOnStatusCode: false },
-      );
+      ).then(response => {
```

Comment:
@npfitz maybe you can help me with this: it's ok for this endpoint to returns 400 here, but I have no idea how to tell cypress to ignore it, seems like `{ failOnStatusCode: false }` doesn't work as exepcted

Reply:
The testing image is... quirky haha. If there are no requests to delete, it'll respond with a 404. I'll update the tests to click "Send a test"